test/library/packages/VectorList/VectorLists.chpl

```diff
@@ -142,8 +142,10 @@ module VectorLists {
   /* Yields all VectorListElement. */
   inline iter VectorList.listElements() {
     var curr = head;
-    do { const next = curr.next; yield curr; curr = next; }
-    while curr != nil;
+    while true {
+      const next = curr.next; yield curr;
```

Comment:
This is test code so I'm not too picky but I don't like these multiple statements per line and think each `;` statement should get its own line.